src/SprykerEco/Zed/AvalaraTaxProductOption/Business/Mapper/AvalaraLineItemMapper.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AvalaraTaxProductOption\Business\Mapper;
+
+use Generated\Shared\Transfer\AvalaraAddressTransfer;
+use Generated\Shared\Transfer\AvalaraLineItemTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\ProductOptionTransfer;
+use Generated\Shared\Transfer\ShipmentTransfer;
+use SprykerEco\Zed\AvalaraTaxProductOption\Dependency\Facade\AvalaraTaxProductOptionToMoneyFacadeInterface;
+
+class AvalaraLineItemMapper implements AvalaraLineItemMapperInterface
+{
+    public const PRODUCT_OPTION_AVALARA_LINE_TYPE = 'cart-item-option';
```

Comment:
* If it is not used outside a class mark as `protected.

Reply:
It is used in tax calculator

Reply:
Resolved.